site/es_MX.all.json

```diff
@@ -421,153 +421,150 @@
 
   "awards_winner_of": { "other": "Ganador de" },
   "donate_button_text": { "other": "Donar" },
-  "wcag_skip_links_header": { "other": "Accessibility Links" },
-  "wcag_skip_links_content": { "other": "Skip to Content" },
 
+  "wcag_skip_links_header": { "other": "Enlaces de accesibilidad" },
+  "wcag_skip_links_content": { "other": "Saltar al contenido" },
 
-  "wcag_homepage_h1": { "other": "Homepage" },
-  "wcag_carousel_h2": { "other": "Carousel" },
-  "wcag_collections_h2": { "other": "Collections" },
+  "wcag_homepage_h1": { "other": "Página principal" },
+  "wcag_carousel_h2": { "other": "Carrusel" },
+  "wcag_collections_h2": { "other": "Colecciones" },
 
-  "wcag_aria_label_footer": { "other": "Footer" },
-  "wcag_aria_label_nav_main": { "other": "Main" },
+  "wcag_aria_label_footer": { "other": "Pie de página" },
+  "wcag_aria_label_nav_main": { "other": "Principal" },
   "wcag_aria_label_nav_sub": { "other": "Sub" },
-  "wcag_aria_label_toggle_nav": { "other": "Toggle Navigation" },
-  "wcag_aria_label_bundle_items": { "other": "Bundle Items" },
-  "wcag_aria_label_carousel": { "other": "Carousel" },
-  "wcag_aria_label_page_collection": { "other": "Page Collection" },
-  "wcag_aria_label_collection_list": { "other": "Collection List" },
-  "wcag_aria_label_collection_slider": { "other": "Collection Slider" },
-  "wcag_aria_label_wishlist": { "other": "Wishlist" },
-  "wcag_aria_label_facebook": { "other": "Share on Facebook" },
-  "wcag_aria_label_twitter": { "other": "Share on Twitter" },
-  "wcag_aria_label_linkedin": { "other": "Share on LinkedIn" },
-  "wcag_aria_label_letterboxd": { "other": "View on Letterboxd" },
+  "wcag_aria_label_toggle_nav": { "other": "Navegación de palanca" },
+  "wcag_aria_label_bundle_items": { "other": "Paquete de artículos" },
+  "wcag_aria_label_carousel": { "other": "Carrusel" },
+  "wcag_aria_label_page_collection": { "other": "Colección de páginas" },
+  "wcag_aria_label_collection_list": { "other": "Lista de colecciones" },
+  "wcag_aria_label_collection_slider": { "other": "Control deslizante de colección" },
+  "wcag_aria_label_wishlist": { "other": "Lista de deseos" },
+  "wcag_aria_label_facebook": { "other": "Compartir en Facebook" },
+  "wcag_aria_label_twitter": { "other": "Compartir en Twitter" },
+  "wcag_aria_label_linkedin": { "other": "Compartir en LinkedIn" },
+  "wcag_aria_label_letterboxd": { "other": "Ver en Letterboxd" },
```

Comment:
I think you've done this better than I did - I translated the company names too which does come out different.

Reply:
I assume these are recognizable in other languages, maybe not Letterboxd